src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -1859,7 +2120,7 @@ struct Bar : IEquatable<Bar>
 {
     private readonly int value;
 
-    public override bool Equals(object obj) => obj is Bar bar && Equals(bar);
+    public override bool Equals(object obj) => false;
```

Comment:
this was hte input side.  this just shuts the test up about using pattern matching when not available.